src/main/java/com/eternalcode/combat/config/implementation/PluginConfig.java

```diff
@@ -105,6 +106,16 @@ public static class Settings {
             EntityDamageEvent.DamageCause.FIRE,
             EntityDamageEvent.DamageCause.FIRE_TICK
         );
+
+        @Description({
+            "# After what type of entity damage should the player get a combat log?",
+            "# You can find a list of all entity types here: https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/entity/EntityType.html",
+            "# If you don't want the combat log to be given to players for a certain entity type, simply remove it from this list"
+        })
+        public List<EntityType> entityTypesToLog = List.of(
+            EntityType.PLAYER,
+            EntityType.ENDER_PEARL
+        );
```

Comment:
samych graczy nie ma co blokować bo to jedyne entity które tagują bezpośrednio lub pośrednio (przez projectile)
ogólnie lepiej odwrócić działanie, tego żeby była to black lista dla "pocisków" (perły strzały itd) `disabledTagEntites`


```suggestion
        @Description({
            "# After what type of projectile entity should not tag the player as fighter?",
            "# You can find a list of all entity types here: https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/entity/EntityType.html"
        public List<EntityType> disabledProjectileEntites = List.of(
            EntityType.ENDER_PEARL
        );
```